Test/Load/inputs/shippingtax_payload.js

```diff
@@ -0,0 +1,12 @@
+import { address } from "./address.js";
+import { create_shippingtax_cart } from "./shippingtax_cart.js";
+
+export function create_shipping_info( order_token, order_reference ) {
+    return {
+        "order_token": order_token,
+        "cart": create_shippingtax_cart( order_reference, false ),
+        "shipping_address": address,
+        "shipping_options": [],
```

Comment:
@kajan-bolt, Can you add more details why we have empty `shipping_options`?
It is not clear for me.

Reply:
This code is supposed to be the request that is sent to the shipping_and_tax webhook that is on the merchant's site. And from what I can see on DataDog, most requests are sent to the shipping_and_tax webhook with an empty `shipping_options` (since the shipping options won't be available to the Bolt Server until we get the response from the shipping_and_tax webhook)